recipes/hyperion/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+python setup.py install
+./configure --prefix=$PREFIX
+make
+make install
```

Comment:
> No, but I've included tests to make sure the executables exist in the test section. I

That is probably enough. I asked just to be certain.

> I was going to include a small test file to run them all to make sure they run (the test would run in a few seconds)

You are the maintainer... Go for it :smile: (Test as much as the CI s can handle!)